### ISSUE TITLE
User Story: FTU_10 Product sorting

### ISSUE THREAD

**Issue by DeeMcCart** [DeeMcCart/CI_PP5_Jeweller#32]
EPIC:  #24 

* FTU_10: As a **First-Time User** I want to **sort my product view by price, date added ....** in order to **see the items most relevant to me**

### Assumptions or Pre-Requisites:
*  Products template is configured and there is sufficient sample data to test
* Sorting critera are agreed e.g. price L->H; price H->L; ????

### Acceptance Criteria: (Must be completed before task is moved to 'Done')
- [x] Sorting is applied to main products view

### Tasks

- [x] Task1 Add sorting html & js on products.html
- [x] Task2 Test/tweak

Before changing task status to 'Review' or 'Done' please provide comment (and screenprints if appropriate) as documentary evidence of task completion

**Comment by DeeMcCart** [comment]
Worked on this 30-31/01/24:  tested ok 31/01 however realised I've forgtten sort by date added (not in models field) and it would actually be nice to have a 'New' tag.... willl park this in 'Review' for now and come back to it:

**Sort by Price:**


![Image](https://github.com/DeeMcCart/CI_PP5_Jeweller/assets/126502176/5d77e0bb-8443-4146-9ede-bac43f2d86c1)


**Sort by :**
**Sort by :**

**Comment by DeeMcCart** [comment]
DMcC 05/02/24:  End of sprint review - only thing is that there is no sorting by date added - will add as a separate Issue #37
